### DISCUSSION TITLE
When To Use Normalization

### DISCUSSION THREAD

**Discussion by vyaspujari** [vdemichev/DiaNN#1011] [General]
I want to perform relative proteomics on samples from mixed species in different ratios (e.g., Sample 1 is 10% human and 90% mouse, and Sample 2 is 50% human and 50% mouse) and am trying to determine whether I should use normalization or analyze these samples separately. The FAQ mentions that normalization should be enabled when considering the "biological variability of interest," but it's not clear what that means in practice. Another example would be, for instance, in a developmental time course, where each time point is a different sample, and I want to perform relative proteomics. Should I input these files together with normalization enabled? If I want to do relative proteomics on such samples, how can I normalize for the total amount that was shot?

**Comment by vdemichev** [comment]
Yes, this will only work well if DIA-NN analyses the samples all together - this is essential.
Not sure if normalisation is going to handle this situation well (too much variation). I would keep it enavbled anyway, and then do some basic correction of normalisation using R/Python, at the protein level.

> how can I normalize for the total amount that was shot?

Usually makes sense to rather select ubiquitously detected proteins and make the median log-levels of those the same across all samples. What would be the point of normalising by the total amount (it is subject to pipetting errors anyway)?
